git/src/org/netbeans/modules/git/ui/repository/remote/RemoteRepository.java

```diff
@@ -686,6 +686,7 @@ public SSHConnectionSettingsType () {
                 settingsPanel.savePasswordCheckBox
             };
             acceptableSchemes = EnumSet.of(Scheme.SSH, Scheme.SFTP);
+            settingsPanel.txtIdentityFile.setText(getDefaultIdentityFilePath());
```

Comment:
This line does not seem necessary. `SSHConnectionSettingsType.populateFields` does call

```java
                String identityFile = getDefaultIdentityFilePath();
                settingsPanel.txtIdentityFile.setText(identityFile);
```

Reply:
Unfortunately populateFields is not called in every case. Most importantly when the repository URL is being copied from github/gitlab where the ssh:// prefix is just missing from the URL. Probably I could raise a bug on that. And mark this line as a workaround in the code, till it gets fixed. I feel the fix would be a bit out of the trivial category.

Reply:
But what if I'm not using an identity file?